tests/random_test.py

```diff
@@ -61,20 +61,24 @@ def _CheckKolmogorovSmirnovCDF(self, samples, cdf):
 
   def _CheckChiSquared(self, samples, pmf):
     alpha = 0.01  # significance level, threshold for p-value
-    values, actual_freq = np.unique(samples, return_counts=True)
+
+    # chi-square test requires sums to be the same, so we over-sample
+    # the range to pick up the full expected distribution.
+    actual_freq = np.bincount(samples, minlength=samples.max() + 100)
```

Comment:
Maybe brief comment on the magic 100 number (we talked in 1:1 and it sounds reasonable).

Reply:
Done